src/components/personalia/Personalia.tsx

```diff
@@ -0,0 +1,64 @@
+import { BodyShort, Label } from "@navikt/ds-react";
+import React from "react";
+import { FormattedDate } from "../FormattedDate";
+import { getAge, getFormattedPersonalia } from "../../personalia.utils";
+import { IPersonalia } from "../../types/personalia.types";
+import { useSanity } from "../../context/sanity-context";
+import { SectionHeading } from "../section/SectionHeading";
+import styles from "./Personalia.module.css";
+
+interface IProps {
+  personalia: IPersonalia;
+}
+
+export function Personalia({ personalia }: IProps) {
+  const { fødselsDato, folkeregistrertAdresse: adresse, kontonummer } = personalia;
+
+  const { getSeksjonTextById, getAppTekst } = useSanity();
+  const textId = "personalia";
+  const personaliaTexts = getSeksjonTextById(textId);
```

Comment:
Lurer på om vi skal bare putte stringen direkte `getSeksjonTextById("personalia);`

Reply:
Jepp, det var det jeg også startet med. Men så så jeg at `<SectionHeading text={personaliaTexts} fallback={textId} />` trenger en fallback. Så dermed endte jeg opp med å legge det i en konstant, siden den refereres to ganger.